generator/src/main/resources/org/javabubble/generator/site/assets/search.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const searchInput = document.getElementById("search-input");
+    const rows = document.querySelectorAll("tbody tr");
+
+    const onSearchInputKeyPress = () => {
+        const searchTextLower = searchInput.value.toLowerCase();
+        rows.forEach((row) => {
+            const namesLower = row.children[0].textContent.toLowerCase();
+            const handlesLower = row.children[1].textContent.toLowerCase();
+            const rowText = namesLower + handlesLower;
```

Comment:
```suggestion
            const rowText = namesLower + "\n" + handlesLower;
```
to avoid direct concatenation. I made a typo in refactoring.